lib/node/runner/api/pagequery.go

```diff
@@ -7,11 +7,15 @@ import (
 	"strconv"
 
 	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/errors"
 	"boscoin.io/sebak/lib/node/runner/api/resource"
 	"boscoin.io/sebak/lib/storage"
 )
 
-const DefaultMaxLimit uint64 = 100
+const (
+	DefaultLimit    uint64 = 100
+	DefaultMaxLimit uint64 = 1000
```

Comment:
`1000` is too big number, I think. `DefaultLimit` is `20` and `DefaultMaxLimit` to `100`, how do you think?

Reply:
I changed it! :->